### PR TITLE
refactor: Move verbose into printer object

### PR DESCRIPTION
**Pull request by TerryHowe**
**What this PR does / why we need it**:

The verbose option should be owned by the printer.

### CODE REVIEW

File: cmd/oras/root/cp_test.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func Test_doCopy(t *testing.T) {
 	opts.From.Reference = memDesc.Digest.String()
 	dst := memory.New()
 	builder := &strings.Builder{}
-	printer := output.NewPrinter(builder)
+	printer := output.NewPrinter(builder, false)
```
Comment (qweeah) [inline cmd/oras/root/cp_test.go:135]: Although it's not used, it's better to set the value to the command option.
```suggestion
	printer := output.NewPrinter(builder, opts.Verbose)
```

File: cmd/oras/root/cp_test.go [hunk 2]
```diff
@@ -156,7 +156,7 @@ func Test_doCopy_skipped(t *testing.T) {
 	opts.Verbose = true
 	opts.From.Reference = memDesc.Digest.String()
 	builder := &strings.Builder{}
-	printer := output.NewPrinter(builder)
+	printer := output.NewPrinter(builder, false)
```
Comment (qweeah) [inline cmd/oras/root/cp_test.go:159]: printer := output.NewPrinter(builder, opts.Verbose)

File: cmd/oras/root/cp_test.go [hunk 3]
```diff
@@ -191,7 +191,7 @@ func Test_doCopy_mounted(t *testing.T) {
 	}
 	to.PlainHTTP = true
 	builder := &strings.Builder{}
-	printer := output.NewPrinter(builder)
+	printer := output.NewPrinter(builder, false)
```
Comment (qweeah) [inline cmd/oras/root/cp_test.go:194]: printer := output.NewPrinter(builder, opts.Verbose)

**Review by qweeah — APPROVED** [review]
LGTM with minor suggestions.

File: cmd/oras/root/cp_test.go (hunk 1 above)
Comment (TerryHowe) [reply cmd/oras/root/cp_test.go]: Yes, done.

**Pull request merged**